com.github.crosswire.Xiphos.json

```diff
@@ -0,0 +1,110 @@
+{
+    "id": "com.github.crosswire.Xiphos",
```

Comment:
They have a website: http://xiphos.org/

So something like `org.xiphos.Xiphos` might be better.

Reply:
OK, fair - I went with where the source code was hosted but I take your point.